harness/determined/keras/_tf_keras_trial.py

```diff
@@ -212,7 +212,7 @@ def _configure_session(
             gpus = tf.config.experimental.list_physical_devices("GPU")
 
             if len(gpus) > 0:
-                local_rank = hvd.local_rank() if hvd_config.use else 0
+                local_rank = hvd.local_rank() if use_horovod else 0
```

Comment:
just use `self.context.distributed.local_rank` here, it'll be easier.

Reply:
this method is a @classmethod, can't use self... methods for class instances.